rules/sinks/third_parties/sdk/musicplayer/javascript.yaml

```diff
@@ -9,5 +9,7 @@ sinks:
     domains:
       - "musicplayer.io"
     patterns:
+      - "music\.163\.com"
```

Comment:
Pattern should be : 
"music\\.163\\.com" or "music[.]163[.]com"

have you run it, is it matching?